arkouda/series.py

```diff
@@ -5,6 +5,7 @@
 import pandas as pd  # type: ignore
 from pandas._config import get_option  # type: ignore
 from typeguard import typechecked
+
```

Comment:
if we add `from __future__ import annotations` to our imports, we won't have to add the quotes around `"Series"` in the typechecking anymore and can just say `Series`

Reply:
Done

Reply:
@pierce314159 This looks good to me now, but leaving for you to double check.